### ISSUE TITLE
ERROR on installation from website extension.gnome.org

### ISSUE THREAD

**Issue by Doctor-Who** [alexispurslane/PRIME-GPU-Profile-Selector#1]
First, many thanks for this precious extension.
Unfortunately, I cannot try it with openSUSE Tumbleweed 

Juste got the icon error when trying installing the extension from gnome website.
Here's my configuration :

```
badwolf@xps-7590:~> neofetch 
OS: openSUSE Tumbleweed  
Host: XPS 15 7590 
Kernel: 6.3.9-1-default 
Resolution: 1920x1080 
DE: GNOME 44.2 
CPU: Intel i7-9750H (12) @ 4.500GHz 
GPU: NVIDIA GeForce GTX 1650 Mobile / Max-Q 
GPU: Intel CoffeeLake-H GT2 [UHD Graphics 630] 
Memory: 5849MiB / 15606MiB 
```
Using X11

**Comment by apusateri** [comment]
Same.

**Comment by APPLESOFT98** [comment]
Same issue on my Acer Aspire E1-570g, I tried installing it even by switching between the iGPU with Wayland and the dGPU with X11, but Gnome Extensions always gives the same error: "GLib.SpawnError: Child Process Execution «prime-select» Failed (File or Directory Does Not Exist)". Strange that says it doesn't find prime-select when I have suse-prime installed on Tumbleweed.

Here's my configuration:
```
OS: openSUSE Tumbleweed x86_64 
Host: Aspire E1-570G V2.06 
Kernel: 6.4.6-1-default 
Shell: bash 5.2.15
DE: GNOME 44.3 
CPU: Intel i3-3217U (4) @ 1.800GHz 
GPU: Intel 3rd Gen Core processor Graphics Controller 
GPU: NVIDIA GeForce 610M/710M/810M/820M / GT 620M/625M/630M/720M 
Memory: 4086MiB / 15870MiB 
```
If I do from terminal `prime-select intel` or `prime-select nvidia` it works, but I would like something more comfortable without going every time in the terminal

**Comment by VortexAcherontic** [comment]
This should be fixed in #2 as soon as it got merged.  
I found this issue to be present on openSUSE MicroOS as well.

The reason it does not work on openSUSE is that this extension does not lookup the actual path of prime-select but rather called the binary directly.
On some systems the binary is located at /usr/sbin which is not in the $PATH of a normal user and therefore can not be found by the extension if not calling the full path.

**Comment by alexispurslane** [comment]
@VortexAcherontic @Doctor-Who @apusateri @APPLESOFT98 Ohmygod I'm _so_ sorry I didn't see this! (I plea extenuating circumstances lol). I realized this might be a problem when I wrote the extension the way I did, but I didn't think it would be an issue because it works on my install of openSUSE Tumbleweed, and I assumed everyone's version of Tumbleweed would install to the same place. I'll get right to work merging the pull request
